.all-contributorsrc

```diff
@@ -0,0 +1,27 @@
+{
+  "projectName": "marvin",
+  "projectOwner": "ajtorres9",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": ["readme.md"],
+  "imageSize": 100,
+  "commit": false,
+  "contributorsPerLine": 7,
+  "badgeTemplate": "  <a href=\"#contributors\"><img alt=\"All Contributors\" src=\"https://flat.badgen.net/badge/all%20contributors/<%= contributors.length %>/orange\"></a>",
```

Comment:
I haven't tested this, but I really hope this works. 🤞

Reply:
Not sure I know how to test this! I guess I'm on the fingers crossed game as well 🤷‍♂️